### ISSUE TITLE
Invalid argument: logits and labels must have the same first dimension, got logits shape [1900,38510] and labels shape [2000]

### ISSUE THREAD

**Issue by gunXT** [variational-attention/tf-var-attention#2]
when i was running the code, i got this warning, not error,  and the predict result was awful after 100 epoch, Any ideas why?

 W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: logits and labels must have the same first dimension, got logits shape [1900,38510] and labels shape [2000]	 [[Node:losses/sequence_loss/SparseSoftmaxCrossEntropyWithLogits/SparseSoftmaxCrossEntropyWithLogits = SparseSoftmaxCrossEntropyWithLogits[T=DT_FLOAT, Tlabels=DT_INT32, _device="/job:localhost/replica:0/task:0/gpu:0"](losses/sequence_loss/Reshape, losses/sequence_loss/Reshape_1/_485)]]
